grudge/discretization.py

```diff
@@ -21,45 +30,55 @@
 """
 
 from pytools import memoize_method
+
 from grudge.dof_desc import (
     DISCR_TAG_BASE, DISCR_TAG_MODAL,
     DTAG_BOUNDARY, DOFDesc, as_dofdesc
 )
+
 import numpy as np  # noqa: F401
-from meshmode.array_context import ArrayContext
-from meshmode.discretization.connection import \
-    FACE_RESTR_INTERIOR, FACE_RESTR_ALL, make_face_restriction
-from meshmode.mesh import BTAG_PARTITION
 
-from warnings import warn
+from arraycontext.context import ArrayContext
 
+from meshmode.discretization.connection import (
+    FACE_RESTR_INTERIOR,
+    FACE_RESTR_ALL,
+    make_face_restriction
+)
+from meshmode.mesh import BTAG_PARTITION
 
-__doc__ = """
-.. autoclass:: DiscretizationCollection
-"""
+from warnings import warn
 
 
 class DiscretizationCollection:
-    """
-    .. automethod :: __init__
+    """A collection of discretizations on various mesh entities
+    (volume, interior facets, boundaries) and associated element
+    groups.
```

Comment:
Maybe mention somehow that all these discretizations are tied to the same underlying mesh?

Reply:
Good suggestion! Added: https://github.com/inducer/grudge/pull/74/commits/52c7318c777efebf2c559986f0f7b6a429898e89